shared/panels.d.ts

```diff
@@ -397,7 +408,8 @@ declare interface Image extends AbstractPanel<'Image'> {
 
 	SetImage(path: string): void;
 
-	SetScaling(mode: 'stretch-to-cover-preserve-aspect' | 'stretch-to-fit-preserve-aspect'): void;
+	/** Valve appear not to have bothered define this in CSS, setting via JS appears to be the only way to set this. */
```

Comment:
It should probably be mentioned that you can set this through xml. (also please check for grammar!!)

Reply:
Sorry for sneaking this in right at the end, but what's wrong with the grammar? Also I was unaware the this could be set through XML, what's the attribute called?